ivy/functional/ivy/nest.py

```diff
@@ -1055,6 +1055,65 @@ def nested_map(
     ret
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
+
+    Examples
+    --------
+    With :class:`Tuple` inputs:
+
+    >>> x = ([[1., 2.], [3., 4.]])
+    >>> function = lambda a : a * 2
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    ([[2., 4.], [6., 8.]])
+
+    With :code:`Dict` input:
+
+    >>> x = {1 : [1, [2, 3]], 2: (4, 5)}
+    >>> function = lambda a : a + 1
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    {1: [1, 2], 2: (4, 5)}
+
+    With :code:`List` inputs:
+
+    >>> x = [['a', 'b', 'c'],
+    ...      ['d', 'e', 'f'],
+    ...      ['g', ['h', 'i']]]
+    >>> function = lambda a: a + 'H'
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    [['aH','bH','cH'],['dH','eH','fH'],['gH',['hH','iH']]]
+
+    >>> x = [[1, 2, 3, 4, 5], [23, 24, 25, 26, 27]]
+    >>> ivy.nested_map(x, include_derived = True)
+    >>> print(x)
+    [[1, 2, 3, 4, 5], [23, 24, 25, 26, 27]]
+
+    >>> nest = ([11, 12, 13, 14, 15],[21, 22, 23, 24, 25])
+    >>> ivy.nested_map(nest, max_depth > 1)
+    >>> print(nest)
+    ([11, 12, 13, 14, 15],[21, 22, 23, 24, 25])
+
+    >>> nest = ([23, 25, 1337], [63, 98, 6])
+    >>> ivy.nested_map(nest, to_mutable = True)
+    >>> print(nest)
+    [[23, 25, 1337], [63, 98, 6]]
+
+    >>> nest = ([3, 5, 7], [6, 9, 10])
+    >>> ivy.nested_map(nest, to_mutable = True, shallow = False)
+    >>> print(nest)
+    [[3, 5, 7], [6, 9, 10]]
+
+    With :class:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([1., 2.]) , b=ivy.array([4., 5.]))
+    >>> function = lambda _: ivy.array([3., 4.])
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    {
+        a: ivy.array([3., 4.]),
+        b: ivy.array([3., 4.])
+    }
```

Comment:
I think you forgot to reorder that. It would be better if it was with the other input types you mentioned above

Reply:
@KareemMAX do you mean it is not necessary to show an 'ivy container' instance for the 'nested_map' function?

Reply:
Nope, I meant that I guess it was suppose to be placed higher up. With `Tuple` and other containers.